test/e2e/agent_test.go

```diff
@@ -61,7 +61,12 @@ spec:
 		SubmitWorkflow().
 		WaitForWorkflow(time.Minute).
 		Then().
-		ExpectWorkflow(func(t *testing.T, _ *metav1.ObjectMeta, status *wfv1.WorkflowStatus) {
+		ExpectWorkflow(func(t *testing.T, meta *metav1.ObjectMeta, status *wfv1.WorkflowStatus) {
```

Comment:
this seems to change the wrong file, I think you need to look at plugin_test.go

Reply:
> this seems to change the wrong file, I think you need to look at plugin_test.go

gitlab ci always showme error here ....  it misunderstood me

Reply:
You're not testing the plugin in this test. You're testing the HTTP template. Having re-reviewed the changes, if you revert this file, I think that we probably don't need a test, because we can add an assertion into the code.